CMakeLists.txt

```diff
@@ -60,7 +60,6 @@ set(CMAKE_CUDA_STANDARD 11)
 if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "PGI")
     message(STATUS "Setting CMAKE_CXX_EXTENSIONS to ON for PGI Compiler")
```

Comment:
Can you please change this so it's only indented 2 spaces to match the rest of the file.

Reply:
Done!